console/servlets/build.gradle

```diff
@@ -0,0 +1,23 @@
+distsDirName = 'webapps'
+
+plugins.apply 'war'
+
+dependencies {
+  providedCompile 'javax.servlet:javax.servlet-api:3.1.0'
+  providedCompile project(':utils:streamscope')
+  providedCompile ext_classpath
+}
+
+war {
```

Comment:
Does all the test case passed? In my case, build failed due to console.war location
Could you tell me if there is something forgotten setting that I missed?